template_upload_file.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<body>
+	<script>
+		function post(path, params, method='post') {
+		  const form = document.createElement('form');
```

Comment:
Inconsistent tabs/spaces used here. Make sure that we always use 4 spaces or 1 tab of 4 spaces.